lib/suspenders/app_builder.rb

```diff
@@ -95,7 +95,7 @@ def enable_rack_deflater
     def setup_asset_host
       replace_in_file 'config/environments/production.rb',
         "# config.action_controller.asset_host = 'http://assets.example.com'",
-        'config.action_controller.asset_host = ENV.fetch("ASSET_HOST", ENV.fetch("HOST"))'
+        'config.action_controller.asset_host = ENV.fetch("ASSET_HOST", ENV.fetch("THIS_HOST"))'
```

Comment:
Line is too long. [95/80]<br>Align the parameters of a method call if they span more than one line.